packages/pds/src/api/app/bsky/feed/getFeed.ts

```diff
@@ -16,24 +17,18 @@ export default function (server: Server, ctx: AppContext) {
         requester,
         feed.view.did,
       )
-      const headers: Record<string, string> = {}
-      const headersToForward = ['accept-language']
-      for (const header of headersToForward) {
-        const value = req.headers[header]
-        if (typeof value === 'string') {
-          headers[header] = value
-        }
-      }
-      const feedOpts = {
-        headers: { ...headers, ...serviceAuthHeaders.headers },
-      }
-      const res = await ctx.appViewAgent.api.app.bsky.feed.getFeed(
+      // forward accept-language header to upstream services
+      serviceAuthHeaders.headers['accept-language'] =
+        req.headers['accept-language']
+      const feedRes = await ctx.appViewAgent.api.app.bsky.feed.getFeed(
         params,
-        feedOpts,
+        serviceAuthHeaders,
       )
+      forwardResHeader(res, feedRes.headers, 'content-language')
```

Comment:
I don't have any qualms with doing it imperatively like this, but on xrpc errors we support a `headers` field that xrpc-server will set on the response.  Eventually we should probably get the same thing going on non-error responses.

Reply:
actually we do have that! I just forgot about it - it should work if we just return headers in the route, next to `body`